services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/controller/UserGroupController.java

```diff
@@ -0,0 +1,245 @@
+package com.sourcefuse.jarc.services.usertenantservice.controller;
+
+import com.sourcefuse.jarc.services.usertenantservice.auth.IAuthUserWithPermissions;
+import com.sourcefuse.jarc.services.usertenantservice.commons.CommonConstants;
+import com.sourcefuse.jarc.services.usertenantservice.dto.Count;
+import com.sourcefuse.jarc.services.usertenantservice.dto.Group;
+import com.sourcefuse.jarc.services.usertenantservice.dto.UserGroup;
+import com.sourcefuse.jarc.services.usertenantservice.enums.RoleKey;
+import com.sourcefuse.jarc.services.usertenantservice.repository.GroupRepository;
+import com.sourcefuse.jarc.services.usertenantservice.repository.UserGroupsRepository;
+import jakarta.validation.Valid;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@Slf4j
+@RequestMapping("/groups")
+@RequiredArgsConstructor
+public class UserGroupController {
+
+  private final GroupRepository groupRepository;
+
+  private final UserGroupsRepository userGroupsRepo;
+
+  @PostMapping("{id}" + "/user-groups")
```

Comment:
```suggestion
  @PostMapping("{id}/user-groups")
```
remove concatenation, create url in single string instead of creating two diff string and then concatenating

Reply:
already done